Sources/Nimble/Matchers/BeAKindOf.swift

```diff
@@ -1,14 +1,22 @@
 import Foundation
 
-#if _runtime(_ObjC)
```

Comment:
Cool, thanks so much for sharing that. In the future, I'll test my changes on Linux like this before pushing.